build/document-extractor.js

```diff
@@ -342,8 +342,17 @@ function _addSingleSpecialSection($) {
     const [proseSections] = _addSectionProse($);
     return proseSections;
   }
-  const query = dataQuery.replace(/^bcd:/, "");
-  const { browsers, data } = packageBCD(query);
+  let query = dataQuery.replace(/^bcd:/, "");
+  let browsers;
+  let data;
+
+  if (dataQuery && dataQuery.trim().match("^http[s]?://")) {
```

Comment:
Could you add a short comment before this block explaining what it does / we need this?